support/Pipelines/Bbh/InitialData.py

```diff
@@ -50,11 +52,14 @@ def id_parameters(
     """
 
     # Sanity checks
-    assert mass_ratio >= 1.0, "Mass ratio is defined to be >= 1.0."
+    mass_ratio = mass_a / mass_b
```

Comment:
If you set this to `max(mass_a, mass_b) / min(mass_a, mass_b)` then `q >= 1` will always be true and you can delete the assert below.